packages/dev/core/src/FlowGraph/flowGraphEngine.ts

```diff
@@ -0,0 +1,36 @@
+import type { Scene } from "../scene";
+import { FlowGraph } from "./flowGraph";
+import { FlowGraphEventCoordinator } from "./flowGraphEventCoordinator";
+
+export class IFlowGraphEngineConfiguration {
+    scene: Scene;

```

Comment:
I assume an engine is a singleton. Which means that you should be able to add graphs that belong to different scenes. 
Would make sense to ask for optional parameters when creating the graph instead of predefining the scene to which the engine belongs to.